src/main/java/com/revature/studyforce/notification/controller/SubscriptionController.java

```diff
@@ -0,0 +1,155 @@
+package com.revature.studyforce.notification.controller;
+
+import com.revature.studyforce.notification.dto.FlashcardSubscriptionDTO;
+import com.revature.studyforce.notification.dto.StacktraceSubscriptionDTO;
+import com.revature.studyforce.notification.model.FlashcardSubscription;
+import com.revature.studyforce.notification.service.FlashcardSubscriptionService;
+import com.revature.studyforce.notification.service.StacktraceSubscriptionService;
+import com.revature.studyforce.notification.service.SubscriptionService;
+import com.revature.studyforce.notification.model.StacktraceSubscription;
+import com.revature.studyforce.notification.model.Subscription;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+
+
+/**
+ * Subscription Controller for
+ *  Subscriptions {@link SubscriptionService}
+ *  FlashcardSubscriptions {@link FlashcardSubscriptionService}
+ *  StacktraceSubscriptions {@link StacktraceSubscriptionService}
+ * @author Brandon Pinkerton
+ */
+@RestController
+@RequestMapping("/subscriptions")
```

Comment:
'@ ResponseBody annotation is unneeded with @ RestController

Reply:
fixed